api/src/clojure/org/akvo/flow_api/endpoint/sync.clj

```diff
@@ -41,20 +41,24 @@
                 db-spec (-> deps :unilog-db :spec (assoc :db-name db-name))
```

Comment:
This function is becoming a monster. Maybe time to clean up?

Reply:
agreed :+1: